src/console/widget.ts

```diff
@@ -360,6 +354,69 @@ class ConsoleWidget extends Widget {
   }
 
   /**
+   * Test whether we should execute the prompt.
+   */
+  private _shouldExecute(): Promise<boolean> {
+    let prompt = this.prompt;
+    let code = prompt.model.source;
+    code = code.slice(0, code.lastIndexOf('\n'));
```

Comment:
Because CodeMirror has already indented the code.  I should turn off auto-indent instead...